questionnaire/2020/questions.yaml

```diff
@@ -161,25 +161,33 @@
     - 10001-50000
     - 50001+
 
-  - title: Which networking vendor devices are you managing?
+  - title: Which vendor devices are you managing?
     type: Multiple choice 
     id: env-vendors
     responses:
+    - Amazon AWS
     - Arista
     - Big Switch
-    - Extreme Networks (brocade)
-    - Cisco IOS/IOS XE
-    - Cisco NX-OS/ACI
+    - Checkpoint
+    - Ciena
+    - Cisco ASA/Firepower
+    - Cisco IOS/IOS XE/Viptela
```

Comment:
Not sure if IOS and Viptela should be grouped together, maybe it's because I'm not familiar with Viptela

Reply:
Cisco released a unified IOS XE software image earlier this year. The same IOS XE is now used for SDWAN (Viptela) and non-SDWAN (traditional) deployments. I think it now makes sense to group them as a single answer.